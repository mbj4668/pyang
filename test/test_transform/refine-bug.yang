module refine-bug {
  namespace "urn:refine-bug";
  prefix "refine-bug";

  grouping stuff {
    leaf-list attr {
      type string;
    }
  }

  container root {
    uses stuff {
      refine attr {
      }
    }
  }
}
