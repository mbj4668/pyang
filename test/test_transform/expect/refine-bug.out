module: refine-bug
  +--rw root
     +--rw attr*   string
